bgpd/bgp_vty.c

```diff
@@ -17700,6 +17811,13 @@ int bgp_config_write(struct vty *vty)
 			vty_endframe(vty, " exit\n");
 		}
 
+		tovpn_sid_index = bgp->tovpn_sid_index;
+		if (CHECK_FLAG(bgp->vrf_flags, BGP_VRF_TOVPN_SID_AUTO)) {
+			vty_out(vty, " sid vpn per-vrf export %s\n", "auto");
```

Comment:
Just `auto` without string interpolation? :)

Reply:
Exactly :) Fixed in https://github.com/FRRouting/frr/pull/11673/commits/b1e7ed5181cef3e08a6dc72e07ca3cc02969292c.